labs/RLSS19_TextWorld.ipynb

```diff
@@ -0,0 +1,2871 @@
+{
```

Comment:
<p>Cool! Maybe an explanation of the difference between <code>admissible_commands</code> and <code>policy_commands</code> is needed here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yfletberliac/rlss2019-hands-on/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Good point, added detailed description of the dict fields</p>